### ISSUE TITLE
Hello Multiply tutorial mistake

### ISSUE THREAD

**Issue by ssolit** [risc0/risc0-rust-examples#54]
In step 2 of the Hello Multiply tutorial, there's a section that says, 

"While we're at it, let's change the rest of the references in factors/src/main.rs. Don't worry about why these lines are included yet; for now, we're just being diligent not to leave dead references behind. Here are what the other two lines with METHOD_NAME_ELF and METHOD_NAME_ID should look like after updating:
```
    let mut prover = Prover::new(MULTIPLY_ELF, MULTIPLY_ID)
        .expect("Prover should be constructed from valid method source code and corresponding image ID");

...

    receipt.verify(MULTIPLY_ID)
        .expect("Code you have proven should successfully verify; did you specify the correct image ID?");
```
"

I believe there is another line you have to change/delete:
```
let method_code = std::fs::read(METHOD_NAME_PATH)
         .expect("Method code should be present at the specified path; did you use the correct *_PATH constant?");
```

**Comment by pdg744** [comment]
Thanks for logging this! 
Tagging @tzerrell and @3lkn

**Comment by tzerrell** [comment]
Thank you for this catch!

There's an underlying issue here: the code in this tutorial uses version 0.13 of the `risc0` crates. In particular it uses `MULTIPLY_ELF` rather than `MULTIPLY_PATH`, which means there is no `std::fs::read` line necessary.

However, this repo and the starter template repo are on version 0.12, and that requires `METHOD_NAME_PATH` instead of `METHOD_NAME_ELF` and hence a `std::fs::read` line as well. We are in the process of migrating the examples into the main RISC Zero repository here: https://github.com/risc0/risc0/tree/main/examples, which is mostly complete, and there the examples are up to date. However, we haven't yet finished the tooling to get the starter template in the https://github.com/risc0/risc0 repo as well; so in the interim we should likely update the starter template repo to 0.13 so we can bring all of the tutorial and what it references into sync with the latest version of our code.

**Comment by tzerrell** [comment]
For the underlying code, our new [`cargo risczero` tool](https://crates.io/crates/cargo-risczero) has been published with 0.14 and should work well as a starting point to get to the contents of the factors example discussed in Hello, Multiply. The remaining work is to update the tutorial to match this, which is an issue for the website, not this repo. This work is included in risc0/website#152, which I've linked back to here to make sure this portion of the work is not overlooked.
